src/server/data-access/MwBotWikibaseRepo.ts

```diff
@@ -0,0 +1,42 @@
+import WikibaseRepo from '@/server/data-access/WikibaseRepo';
+import mwbot from 'mwbot';
+import TechnicalProblem from '@/server/data-access/error/TechnicalProblem';
+import EntityNotFound from '@/server/data-access/error/EntityNotFound';
+import Entity from '@/store/entity/Entity';
+
+export default class MwBotWikibaseRepo implements WikibaseRepo {
+	private bot: mwbot;
+
+	public constructor( bot: mwbot ) {
+		this.bot = bot;
+	}
+
+	public getEntity( id: string ): Promise<Entity> {
```

Comment:
This might even return an actual `FingerprintableEntiy`.

Reply:
I had this first but in the end decided to take the potential failure to convert the (found) API entity into a FingerprintableEntity (happening in EntityInitializer) out of this and put it into the layer next up (to not let this class have too many responsibilities). Background being that the entity could be found by the API just fine and our EntityInitializer just lost touch with the format they come in. What do you think about that? Consequently we'd need a transport vehicle for the moment in-between (strictly speaking even if it were within MwBotWikibaseRepo).
That this uses an `Entity` interface that is part of store is indeed utter bollocks - briefly looking at the history revealed that my mental model of "Entity" (and what I assumed auto-completed) referred to what was formerly known as `@/common/interfaces/EntityType` and has been discontinued.

Reply:
I now changed the interface to have a `getFingerprintableEntity()` method (instead of `getEntity()`) and have it resolve to `FingerprintableEntity`. I'm not a fan but the name should also be enough of a red flag to remind us that some further definition is due.